Update uc-survey.md (#129)
solid/authorization-panel: proposals/wac-ucr/uc-survey.md

some rational ways to do this using machinery we already have.
* +1 elf-pavlik:
* 0 csarven: Too specific. May require querying. Alternative based on simpler use cases: hide references to resources from a container's index from agents that do not have read permission. [sw?]

> * +1 jaxoncreed:* 

Reviewer: ```suggestion
* +1 jaxoncreed:
```

piloted scenarios)
* 0 csarven: it depends on the environment of the application.
* +1 jaxoncreed:
* im a little unclear on this case

### Limiting application access while not acting as resource controller
URL: https://solid.github.io/authorization-panel/wac-ucr/#uc-client-constraints

* 0 csarven: A bit of a low-level server-side plumbing. Unclear how an application (like [ap]) may want/need to set a policy as such.

> * im a little unclear on this case

Reviewer: ```suggestion
* KaiGilb: im a little unclear on this case
```

* +1 elf-pavlik:
* +1 csarven: Duplicate of #basic-readonly.
* +1 jaxoncreed:
* +1 KaiGilb: graphMetrix

### Limiting access to other authorization conditions
URL: https://solid.github.io/authorization-panel/wac-ucr/#uc-historyofchanges

* 0 csarven: Complex use case. Perhaps change focus to auxiliary resources.
* +1 jaxoncreed:
* +1 KaiGilb: graphMetrix. Im not sure this should be handled by permission, but yes, they should not see it.

### Minimal Credential Disclosure
URL: https://solid.github.io/authorization-panel/wac-ucr/#uc-minimalcredentials

* +0 justinwb: This scenario is legitimate though I believe we'll likely
get to VC in the next cycle.
* +1 elf-pavlik:

> * +0 Kai Gilb: graphMetrix Sound good. I would again think this would be app controlled

Reviewer: ```suggestion
* +0 KaiGilb: graphMetrix Sound good. I would again think this would be app controlled
```

specifically wants the information to be divulged.
* +1 csarven: Wide use. I don't plan to implement it.
* +1 jaxoncreed:

> * this seems like a duplicate from 2 up

Reviewer: ```suggestion
* KaiGilb: this seems like a duplicate from 2 up
```

* +1 elf-pavlik:
* +1 csarven: I don't plan to implement it.
* +1 jaxoncreed:

> * +0 Kai Gilb: graphMetrix. Sounds good for the future, we are not there quite yet.

Reviewer: ```suggestion
* +0 KaiGilb: graphMetrix. Sounds good for the future, we are not there quite yet.
```

* +1 elf-pavlik:
* +1 csarven: Wide use. I don't plan to implement it.
* +1 jaxoncreed:

> * +0 Kai Gilb: graphMetrix

Reviewer: ```suggestion
* +0 KaiGilb: graphMetrix
```
